vmm/src/vm.rs

```diff
@@ -2440,6 +2485,7 @@ impl Vm {
 impl Pausable for Vm {
     fn pause(&mut self) -> std::result::Result<(), MigratableError> {
         event!("vm", "pausing");
+        info!("Pause VM for migration");
```

Comment:
This log may not be right, 'cause vm may pause from api request also.

Reply:
The new version has fixed this problem.

Thanks